app/src/main/res/layout/time_dialog.xml

```diff
@@ -50,19 +50,46 @@
             android:visibility="gone">
 
         <TextView
-                android:text="00:00:00"
-                android:layout_gravity="center"
-                android:textSize="32sp"
-                android:textColor="?android:attr/textColorPrimary"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:id="@+id/time"/>
+            android:text="00:00:00"
+            android:layout_gravity="center"
+            android:textSize="32sp" 
+            android:textColor="?android:attr/textColorPrimary"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/time"/>
 
         <Button
-                android:text="@string/disable_sleeptimer_label"
-                android:layout_width="match_parent"
+            android:text="@string/disable_sleeptimer_label"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:id="@+id/disableSleeptimerButton"/>
+            
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
```

Comment:
Wouldn't `wrap_content` without `layout_weight` be easier here? Or am I overlooking something?

Reply:
Then the buttons would all have different sizes, which I wanted to prevent. What do you think?

Reply:
Do you mean width or height of the buttons? For the width, I think it should be something like `weight_sum` here instead of `layout_weight`.

Reply:
I ment the width. Android Studio suggested that I should set the height to 0dp, as I'm using layout_weight.

But I'm not too familar with Android anymore, it's. been over three years for me so maybe you could add this as code suggestion directly? That would be helpful.